roles/yoda_test/defaults/main.yml

```diff
@@ -93,6 +93,9 @@ test_group_users:
   - {groupName: priv-group-add, user: functionaladminpriv,            role: manager}
   - {groupName: priv-group-add, user: functionaladmingroup,           role: manager}
 
+  - {groupName: priv-admin, user: functionaladminpriv,               role: manager}
+  - {groupName: priv-category-add, user: functionaladmincategory,           role: manager}
```

Comment:
This line is redundant and can be removed, user is also added on l91.

Reply:
Sure